manifests/defaults.pp

```diff
@@ -16,7 +16,7 @@
   if $create_base_rules {
     $rules.each |$rule_name, $params| {
       if !defined(Logrotate::Rule[$rule_name]) {
-        $_merged_params = merge($rule_default,$params)
+        $_merged_params = stdlib::merge($rule_default,$params)
```

Comment:
Does the lower band of stdlib in metadata need to be changed ?

Reply:
stdlib::merge would require stdlib9. But it occurs to me that a better solution might be to use the + operator and avoid the whole issue.